src/App.css

```diff
@@ -141,27 +139,34 @@ a:hover {
 }
 
 .coins-container {
-  flex-direction: column;
-  gap: 1rem;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 50% 47%;
+  margin-bottom: 0.5rem;
   align-items: center;
   justify-content: center;
-  width: 100%;
-  padding-bottom: 1rem;
 }
 
 .card-container {
-  width: 74%;
+  width: 100%;
+  height: 7rem;
+  margin-top: 0.02rem;
+  margin-bottom: 0.1rem;
+  margin-right: 0.5rem;
 }
 
 .coin-card {
-  gap: 1rem;
-  justify-content: center;
+  column-gap: 0.02rem;
+  justify-content: left;
   align-items: center;
-  padding: 1rem;
-  background-color: rgb(46 47 53 / 80.1%);
+  padding-left: 0.5rem;
```

Comment:
- [ ] great job making it grid. please like the previous reviwer noted, kindly try to follow the design guideline including the Layout: (composition and space between elements. ). please add the sizes of the cards and make their margins/borders thin and light like in the original design. 

yours              |  original
:-------------------------:|:-------------------------:
![Screenshot (995)_LI](https://user-images.githubusercontent.com/65298048/234491570-ec0720ad-063b-42ac-a10b-031d2d9723cb.jpg)  |  ![Screenshot (996)](https://user-images.githubusercontent.com/65298048/234491595-e8892902-de49-4639-b032-a96278b3d294.png)


- [optional] please, you can add the arrows on the cards and the thick and light colors, so it matches well with the design.

Reply:
If I were given one minute to check someone's PR then there is always room for improvements  i have seen worse in design and logic  projects got approved than this one ,,,,.but i am very much preplexed about the rules you follow while checking PRs of trainees